### PR TITLE
Moved IPCS related data to projection solver class

### LINKED ISSUE #127: [master] Move IPCS specific data from timestepping class to IPCS solver

### PR DESCRIPTION
**Pull request by sebglane**
Fixes #127

### CODE REVIEW

**Comment by sebglane** [comment]
No testing performed yet. After merging #124 to `master` we could merge `master` to `move_ipcs_related_data` and run the automatic testing...

**Review by j507 — APPROVED** [review]
Good to go as long as the changes do not change the results of a convergence test for example

File: include/rotatingMHD/navier_stokes_projection.h
Comment (sebglane) [reply include/rotatingMHD/navier_stokes_projection.h]: Fixed in d0d73ef

**Comment by sebglane** [comment]
Output of the DFG benchmark differs from reference. I will take a look at this issue tomorrow.

**Comment by sebglane** [comment]
The output differs because the value `previous_alpha_zeros` are wrong in the current master branch, see output below.
```
Solving until t = 350...
Step =      0, Current time = 0.0e+00, Next time step = 2.5e-02
alpha: 1.0e+00, -1.0e+00, 0.0e+00, 
previous_alpha_zeros: 1.5e+00, 1.0e+00, 
previous_step_sizes: 0.0e+00, 0.0e+00, 
Step =      1, Current time = 2.5e-02, Next time step = 2.5e-02
    dp = +8.585950e+01 C_d = +1.337508e+02 C_l = -2.073725e-01
alpha: +1.5, -2, +0.5, 
previous_alpha_zeros: +1.5, +1.5, 
previous_step_sizes: +0.025, +0, 
Step =      2, Current time = +5.000000e-02, Next time step = +2.500000e-02
    dp = +2.674982e+01 C_d = +5.115431e+01 C_l = -1.540453e-01
alpha: +1.5, -2, +0.5, 
previous_alpha_zeros: +1.5, +1.5, 
previous_step_sizes: +0.025, +0.025, 
```

**Pull request merged**
